src/js/layout.js

```diff
@@ -76,3 +76,19 @@ export function raiseInputLabel() {
     });
   });
 }
+export function smoothScrolling() {
```

Comment:
Please move it to separate file

- [ ] separate file

Reply:
`utils.js` maybe?

Reply:
or `smoothScrolling.js`, becuase this is the only one